gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/ExecutionContext.java

```diff
@@ -232,10 +234,12 @@ private void buildSourceToExpressionMaps() {
       Expression expression = orderByExpression.getExpression();
       Map<String, List<Expression>> map =
           getDataSourceToExpressionMap(Collections.singletonList(expression));
-      // There should only be one element in the map.
```

Comment:
The assumption was wrong. The map could have more elements, if an attribute is present in multiple sources. Does not affect anything at all